### DISCUSSION TITLE
[@types/node] FSWatcher ref and unref are missing

### DISCUSSION THREAD

**Discussion by JohnnyBill** [DefinitelyTyped/DefinitelyTyped#68296] [Issues with a @types package] [answered]
Calling fs.watch returns an FSWatcher instance.

The methods `ref` and `unref` are missing in the typings.

https://nodejs.org/docs/latest/api/fs.html#watcherref

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment, marked as answer]
I have created PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68300
